configure.ac

```diff
@@ -182,7 +186,7 @@ case $host in
   *linux*)
     BUILD_OS=linux
 
-    if test "x$build_client" = "xyes"; then
+    if test "x$build_clientx$with_gui" = "xyesxyes"; then
```

Comment:
Is this valid? I've never seen this format before. Kinda cool - albeit a bit confusing - if it is.